### ISSUE TITLE
HookWebpackError: plugins is not iterable

### ISSUE THREAD

**Issue by xavierfoucrier** [cascornelissen/svg-spritemap-webpack-plugin#164]
Hello @cascornelissen,

I have another issue with the latest version of `svg-spritemap-webpack-plugin@4.0.1`.
When you omit the `plugins` parameter of SVGO 2, webpack throw a `HookWebpackError`.

Using **an empty array** for plugins like `plugins: []` works.

It should be fine to avoid this parameter if a developer just want to apply a global option, like `floatPrecision`, and use the default plugins configuration.

Thanks for the feedback! :wink: 

> You can update/fetch the previous repository I share with you to see the issue.

**Comment by cascornelissen** [comment]
Thanks for the issue, this should be fixed in a5d1705. Would you mind verifying by installing via GitHub before I create a release and publish it?

```
npm install cascornelissen/svg-spritemap-webpack-plugin#master
```

**Comment by xavierfoucrier** [comment]
@cascornelissen sure, I am on it :wink:

**Comment by xavierfoucrier** [comment]
@cascornelissen looks like it works fine :+1:

**Comment by cascornelissen** [comment]
Great, just published [`4.0.2`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v4.0.2) with this fix, thanks once more for reporting! ❤️
